src/config/chains.ts

```diff
@@ -23,3 +33,16 @@ export function defaultGetRPCUrl(chainId: number): string {
   }
   return chains[chainId].rpcUrl
 }
+
+// Helper function to get asset weight (defaults to 1 if not specified)
+export function getAssetWeight(chainId: number, symbol: string): number {
+  if (
+    !chains[chainId] ||
+    !chains[chainId].assets ||
+    !chains[chainId].assets[symbol]
```

Comment:
would something like this work too? `!chains[chainId]?.assets?.[symbol]`

Reply:
yep! cleaner thanks